### ISSUE TITLE
XLM-USD bot tried to buy and received "insufficient funds" from the coinbase API

### ISSUE THREAD

**Issue by barryw** [whittlem/pycryptobot#187]
**Describe the bug**
Coinbase returned an "insufficient funds" error when my XLM-USD bot tried to buy.

**To Reproduce**
Not sure. Feels like an edge case.

**Expected behavior**
Expected to detect this error. Coinbase returns a 400, but the payload probably contains a message indicating insufficient funds. Because of this, the bot set its last status to "BUY" even though it held no XLM.

**Screenshots**
 2021-05-14 07:45:00 | XLM-USD 900 | Close: 0.68 | BUY

XLM balance before order: 0.0
USD balance before order: 255.95
error: POST (400) https://api.pro.coinbase.com/orders - Insufficient funds
XLM balance after order: 0.0
USD balance after order: 255.95

**Comment by Pulcow** [comment]
got the same error here, only on the XLM-EUR market

` 2021-05-14 16:15:00 | XLM-EUR 900 | Close: 0.56 | BUY

XLM balance before order: 0.0
EUR balance before order: 51.53
error: POST (400) https://api.pro.coinbase.com/orders - Insufficient funds
XLM balance after order: 0.0
EUR balance after order: 51.53`

**Comment by Pulcow** [comment]
i got the same problem here, i got 5 instances actually and XLM-EUR market is the only one with this problem.

```
 2021-05-14 16:15:00 | XLM-EUR 900 | Close: 0.56 | BUY

XLM balance before order: 0.0
EUR balance before order: 51.53
error: POST (400) https://api.pro.coinbase.com/orders - Insufficient funds
XLM balance after order: 0.0
EUR balance after order: 51.53
```

**Comment by horamarques** [comment]
Related with:
https://github.com/whittlem/pycryptobot/issues/185

I've suggested a possible solution for this..

**Comment by iThom** [comment]
Hello,
This issue should be solved now with the merged PR #194 
@barryw @Pulcow Can you confirm it ? Did you still get this error for XLM ?
